### PR TITLE
Safety nets

### PR DESCRIPTION
**Pull request by shortstuffsushi**
@qinweigong

### CODE REVIEW

File: Branch-SDK/Branch-SDK/BNCLinkData.m [hunk 1]
```diff
@@ -131,16 +115,6 @@ - (void)setupParams:(NSDictionary *)params {
     }
 }
 
-- (void)setObject:(id)anObject forKey:(id <NSCopying>)aKey {
```
Comment (qinweigong) [inline Branch-SDK/Branch-SDK/BNCLinkData.m]: Isn't this method still called? Branch.m line 896, BranchShortUrlRequest.m line 79 and BranchShortUrlSyncRequest.m line 73?

Comment (shortstuffsushi) [inline Branch-SDK/Branch-SDK/BNCLinkData.m]: Nope, that uses the `BNCLinkCache` methods, which aren't looking for `NSCopying`, just that your item is a `BNCLinkData` item.

Comment (qinweigong) [inline Branch-SDK/Branch-SDK/BNCLinkData.m]: oh right!

**Comment by qinweigong** [comment]
:+1:

**Pull request merged**